tests/string_test.cpp

```diff
@@ -475,12 +475,16 @@ TEST_CASE("StringView", "")
 
 TEST_CASE("Trim", "")
 {
+	REQUIRE(bx::strLTrim("a", "a").isEmpty() );
+	REQUIRE(bx::strRTrim("a", "a").isEmpty() );
+	REQUIRE(bx::strTrim("a", "a").isEmpty() );	
+
 	REQUIRE(0 == bx::strCmp(bx::strLTrim("abvgd", "ab"), "vgd") );
-	REQUIRE(0 == bx::strCmp(bx::strLTrim("abvgd", "vagbd"), "abvgd") );
+	REQUIRE(0 == bx::strCmp(bx::strLTrim("abvgd", "vagbd"), "") );
```

Comment:
This should not trim anything. Should be "abvgd", because string "vagbd" doesn't match anything from the left side. RHS string should not be treated as individual characters.

Reply:
Actually my bad... It's good.